src/transport/SecureSession.h

```diff
@@ -311,7 +317,9 @@ class SecureSession : public Session, public ReferenceCounted<SecureSession, Sec
     /// Timestamp of last rx. @see ActiveTimestamp in the spec
     System::Clock::Timestamp mLastPeerActivityTime = System::SystemClock().GetMonotonicTimestamp();
 
-    ReliableMessageProtocolConfig mRemoteMRPConfig = GetDefaultMRPConfig();
+    ReliableMessageProtocolConfig mRemoteMRPConfig   = GetDefaultMRPConfig();
+    System::Clock::Timestamp mRemoteMRPIdleTimeout   = GetRetransmissionTimeout(mRemoteMRPConfig.mIdleRetransTimeout);
+    System::Clock::Timestamp mRemoteMRPActiveTimeout = GetRetransmissionTimeout(mRemoteMRPConfig.mActiveRetransTimeout);
```

Comment:
Adding these members adds a nontrivial amount of storage, because we have a session pool.  Why can't we just compute the value in `GetAckTimeout` instead of storing it?

Reply:
I thought that is would have been better to not have to calculate them all the time. But I have not though that we have a session pool and that will make a size increase. I have updated the code to do that dynamically.